pegasus/migrations/114_change_contact_rollups_email_collation.rb

```diff
@@ -0,0 +1,7 @@
+Sequel.migration do
+  # change the contact_rollups email column to utf8_bin collation order. This is so queries from this table sorted by
+  # email get sorted in the exactly the same alphabetization convention as Ruby <=> operator, which is important
+  # for contact sync process.
+  up { run 'ALTER TABLE contact_rollups MODIFY email VARCHAR(255) COLLATE utf8_bin' }
```

Comment:
As far as I can tell, in order to change column collation in a Sequel migration you have to use raw SQL as I have here.